wai-aria/scripts/aria-utils.js

```diff
@@ -138,8 +138,19 @@ const AriaUtils = {
       promise_test(async t => {
         const expectedLabel = el.getAttribute("data-expectedlabel");
         let computedLabel = await test_driver.get_computed_label(el);
-        // Todo: Remove whitespace normalization after https://github.com/w3c/accname/issues/192 is addressed. Change prior line back to `const`, too.
-        computedLabel = computedLabel.trim()
+
+        // See:
+        // - https://github.com/w3c/accname/pull/165
+        // - https://github.com/w3c/accname/issues/192
+        // - https://github.com/w3c/accname/issues/208
+        //
+        // AccName references HTML's definition of ASCII Whitespace
+        // https://infra.spec.whatwg.org/#ascii-whitespace
+        // which matches tab (\t), newline (\n), formfeed (\f), return (\r), and regular space (\u0020).
+        // but it does NOT match non-breaking space (\xA0,\u00A0) and others matched by \s
+        const asciiWhitespace = /\t\n\f\r\u0020+/g;
+        computedLabel = computedLabel.replace(asciiWhitespace, '\u0020').trim();
```

Comment:
`trim` operates on [the same characters](https://tc39.es/ecma262/#sec-trimstring) as [RegExp's `\u` character class](https://tc39.es/ecma262/#sec-compiletocharset).

```suggestion
        computedLabel = computedLabel.replace(asciiWhitespace, '\u0020')
          .replace(/^\u0020|\u0020$/g, '');
```

Reply:
Rather, RegExp's `\s` character class.

Reply:
```suggestion
        computedLabel = computedLabel.replace(asciiWhitespace, '\u0020').replace(/^\u0020|\u0020$/g, '');
```